.github/workflows/enterprise-pr-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Enterprise PR check
+on:
+  pull_request:
+    branches:
+      - main
+      - 'v[0-9]+.[0-9]+.x'
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v2
```

Comment:
I'd say that the rule of thumb is to try and checkout the code for this marketplace feature so we can be able to know that it doesn't do anything funny!
(The chances are too few, but we always want to be on the safe side!)

Reply:
It seems good to me but I'm not GH actions expert. We can also move it to our own GH actions if we prefer as it's quite simple to implement (only one GH API call).